bin/Debug/nightOwl.exe.config

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
every file from bin should be ignored by gitignore - there is no point of adding it, because then repo is heavier, and bin is generated everytime you build your application, so there is no need to share these files